examples/build_mixed_oxide.py

```diff
@@ -0,0 +1,17 @@
+
+'''
+Testing the build_mixed_oxide functionality
+'''
+
+def test_build_oxide():
+    from carmm.build.mixed_oxide import mixed_oxide
+    from ase.io import read, write
+    import numpy as np
+    orig_struc = read('/data/mixed_oxide_files/geometry.in')
+    mixed_oxide_geometry = mixed_oxide(orig_struc,'Co', 'Mn',2,+2,+2, 3, 5)
+
+    chem_sym = np.array(mixed_oxide_geometry.get_chemical_symbols())
+    unique, counts = np.unique(chem_sym, return_counts=True)
+    assert counts[list(unique).index('Mn')]==10
```

Comment:
Can you add more assert tests to check different parts of the new structure?
e.g. Mn charges and magnetic moments, maybe an explicit neutrality test here too to check it hasn't skipped over it somehow?

Reply:
If the neutrality check is not passed, the assert statement here will give an error and hence the neutrality check is already encompassed. I don't understand what you mean by different parts of the new structure?

Reply:
What I mean is that I think it's beneficial to test lots of the individual capabilities of a function, so that if someone accidentally changes a small part of the code, it gets picked up.
e.g. Currently, if someone were to accidentally change the code putting the new charges into the new atoms object, this test script wouldn't pick that up. All your test asks is whether or not there are 10 Mn atoms, not whether their charges match up with what they should be.
Same with magnetic moments.

Reply:
In the case of the neutrality test, I think you should still have an explicit test for it.
Correct me if I'm wrong, but if the neutrality test fails while it's testing, surely all you will see from the failure output is that it failed because there weren't 10 Mn atoms, which is misleading.
Or would it output the neutrality test error?